receiver/otelarrowreceiver/config.go

```diff
@@ -25,14 +38,10 @@ type ArrowConfig struct {
 	// passing through, they will see ResourceExhausted errors.
 	MemoryLimitMiB uint64 `mapstructure:"memory_limit_mib"`
 
-	// AdmissionLimitMiB limits the number of requests that are received by the stream based on
-	// request size information available. Request size is used to control how much traffic we admit
-	// for processing, but does not control how much memory is used during request processing.
+	// Deprecated: This field is no longer supported, use cfg.Admission.AdmissionLimitMiB instead.
 	AdmissionLimitMiB uint64 `mapstructure:"admission_limit_mib"`
```

Comment:
Please name this field `deprecatedAdmissionLimitMiB` and prevent external references in the code, while it still allows a field named "admission_limit_mib" to be parsed and used (via a deprecated symbol). Likewise for `deprecatedWaiterLimit`.

Reply:
Just a note, due to how fields are parsed through `mapstructure`, the variable name is required to be public here so that `mapstructure` can properly parse the provided config option value into the `golang` variable.

Maybe keep the deprecated prefix but just capitalize it?

Reply:
Thanks for the review! Changed the names to `DeprecatedAdmissionLimitMiB` and `DeprecatedWaiterLimit`